testing/pom.xml

```diff
@@ -377,6 +377,10 @@
                   <goal>verify</goal>
                 </goals>
                 <configuration>
+                  <systemPropertyVariables>
```

Comment:
Where did this come from?

Reply:
I also don't think we should make this change, I guess it was on purpose to have more than one node there.. Not sure..